src/cljs/rems/fields.cljs

```diff
@@ -271,19 +294,13 @@
       [add-symbol]
       " "
       (text :t.form/upload)]
-     [:a.upload-info-button.btn.btn-link
-      {:data-toggle "collapse"
-       :href (str "#" info-id)
-       :aria-label (text :t.form/upload-extensions)
-       :aria-expanded "false"
-       :aria-controls info-id}
-      [:i.fa.fa-question-circle]]
-     [:div.upload-info.collapse {:id info-id
-                                 :ref focus-when-collapse-opened
-                                 :tab-index "-1"}
-      [text :t.form/upload-extensions]
-      ": "
-      attachment-types/allowed-extensions-string]]))
+     [collapse
+      info-id
+      (text :t.form/upload-extensions)
+      focus-when-collapse-opened
+      [:span [text :t.form/upload-extensions]
+       ": "
+       attachment-types/allowed-extensions-string]]]))
```

Comment:
Here you're passing a non-string `body-text` argument to `collapse`. Perhaps linkifying the body shouldn't be in `collapse` but instead the caller. We only use linkify for a couple of things but we use collapse everywhere.